frontend/packages/dsco/src/divider/Divider.tsx

```diff
@@ -0,0 +1,38 @@
+import classNames from 'classnames';
+import {HTMLAttributes} from 'react';
+
+import moduleStyles from './divider.module.scss';
+
+export interface DividerProps extends HTMLAttributes<HTMLElement> {
+  /** Divider Color*/
+  color?: 'primary' | 'strong';
+  /** Margin */
+  margin?: 'none' | 'small' | 'medium' | 'large' | 'extra-large';
+}
+
+/**
+ * ### Production-ready Checklist:
+ * * (✔) implementation of component approved by design team;
+ * * (✔) has storybook, covered with stories and documentation;
+ * * (✔) has tests: test every prop, every state and every interaction that's js related;
+ * * (see apps/test/unit/componentLibrary/Divider.test.tsx)
```

Comment:
nit: this is an old route to the tests folder, right now it will be 
`./__tests__/Divider.test.tsx`